modules/nf-core/tcoffee/tcs/main.nf

```diff
@@ -0,0 +1,74 @@
+process TCOFFEE_TCS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-a76a981c07359a31ff55b9dc13bd3da5ce1909c1:84c8f17f1259b49e2f7783b95b7a89c6f2cb199e-0':
+        'biocontainers/mulled-v2-a76a981c07359a31ff55b9dc13bd3da5ce1909c1:84c8f17f1259b49e2f7783b95b7a89c6f2cb199e-0' }"
+
+    input:
+    tuple val(meta), path(msa)
+    tuple val(meta2), path(lib)
```

Comment:
Why 2 input channels? It seems meta2 is not even used in the script

Reply:
You can optionally pass a lib file. And if you doyou pass it through a channel that created such library in an alignment step before, so it will have it is own meta anyway. It is not used because not needed. But it will be passed in case with the lib file.

Reply:
Okay, should be fine as is